src/test/java/org/jenkinsci/plugins/github_branch_source/EndpointTest.java

```diff
@@ -90,6 +91,13 @@ public void canPostAsAdmin_doCheckApiUri() throws Exception {
         assertTrue(TestRoot.get().visited);
     }
 
+    @Test
+    @Issue("JENKINS-73053")
+    public void manageCanSetupEndpoints() throws Exception {
+        HtmlPage htmlPage = j.createWebClient().login("alice").goTo("manage/configure");
+        assertTrue(htmlPage.getVisibleText().contains("GitHub Enterprise Servers"));
```

Comment:
Does not verify that you can actually save new endpoints, but that is much harder to test in WebClient (given YUI craziness), and I guess ought to be implied by the same `getRequiredGlobalConfigPagePermission`. Did you at least try it interactively? (Most easily by just setting a breakpoint in this test and then logging in as `alice` / `alice` from a real browser.)

Reply:
Verified manually. It works as expected.